generators/server/files.js

```diff
@@ -142,7 +142,11 @@ function writeFiles() {
             }
 
             // Thymeleaf templates
-            this.copy(SERVER_MAIN_RES_DIR + 'templates/error.html', SERVER_MAIN_RES_DIR + 'templates/error.html');
+            if (this.angularVersion === 'angular1') {
+                this.copy(SERVER_MAIN_RES_DIR + 'templates/error.html', SERVER_MAIN_RES_DIR + 'templates/error.html');
+            } else {
+                this.copy(SERVER_MAIN_RES_DIR + 'templates/error.component.html', SERVER_MAIN_RES_DIR + 'templates/error.component.html');
```

Comment:
Sorry @deepu105, so can you quickly explains me what `test-creation.js` and `files.js` are for then ?